src/datalogger/src/drivers/resources/gpio.rs

```diff
@@ -46,7 +46,7 @@ pub struct GpioRequest {
     gpio6 : bool,
     gpio7 : bool,
     gpio8 : bool,
-    usart : bool,
+    pub usart : bool,
```

Comment:
implement a getter for this rather than making the variable pub, since the value should be able to be mutated after initialization.